packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerWithHighlight/PdfViewerWithHighlight.stories.tsx

```diff
@@ -328,6 +332,8 @@ storiesOf('DocumentPreview/components/PdfViewerWithHighlight', module)
         highlights={highlightKnob.data[highlights]}
         activeIds={activeIds}
         setCurrentPage={handleSetCurrentPage}
+        // pulled from pdfjs-dist (see main.js > staticDirs)
+        pdfWorkerUrl={'pdf.worker.min.js'}
```

Comment:
Nit: can we declare this url at the top of the file and pass it in as a var everywhere else?